Packaging for bibutils is maintained in git; this diff represents
several git commits.

Please consider "debcheckout bibutils" in order to send patches.
--- bibutils-4.12.orig/bibutils.dbk
+++ bibutils-4.12/bibutils.dbk
@@ -6,11 +6,11 @@
   <!ENTITY dhfirstname "<firstname>David</firstname>">
   <!ENTITY dhsurname   "<surname>Bremner</surname>">
   <!-- Please adjust the date whenever revising the manpage. -->
-  <!ENTITY dhdate      "<date>February 10, 2008</date>">
+  <!ENTITY dhdate      "<date>December 10, 2010</date>">
   <!-- SECTION should be 1-8, maybe w/ subsection other parameters are
        allowed: see man(7), man(1). -->
   <!ENTITY dhsection   "<manvolnum>1</manvolnum>">
-  <!ENTITY dhemail     "<email>bremner@unb.ca</email>">
+  <!ENTITY dhemail     "<email>bremner@debian.org</email>">
   <!ENTITY dhusername  "David Bremner">
   <!ENTITY dhucpackage "<refentrytitle>BIBUTILS</refentrytitle>">
   <!ENTITY dhpackage   "bibutils">
@@ -18,14 +18,14 @@
   <!ENTITY debian      "<productname>Debian</productname>">
   <!ENTITY gnu         "<acronym>GNU</acronym>">
   <!ENTITY gpl         "&gnu; <acronym>GPL</acronym>">
-  <!ENTITY version	"3.40">
+  <!ENTITY version	"4.12">
 
   ]>
 <refentry>
   <refentryinfo>
     <address>&dhemail;</address>
     <copyright>
-      <year>2008</year>
+      <year>2010</year>
       <holder>&dhusername; (Manual)</holder>
     </copyright>
     <copyright>
@@ -107,6 +107,13 @@
           </seg>
           <seg>convert bibtex to MODS</seg>
       </seglistitem>
+	<seglistitem>
+          <seg>
+            <xref linkend="biblatex2xml" />
+          </seg>
+          <seg>convert biblatex to MODS</seg>
+      </seglistitem>
+
       <seglistitem>
         <seg>
           <xref linkend="copac2xml" />
@@ -117,6 +124,12 @@
       </seglistitem>
       <seglistitem>
         <seg>
+          <xref linkend="ebi2xml" />
+        </seg>
+        <seg>convert EBI XML to MODS</seg>
+      </seglistitem>
+      <seglistitem>
+        <seg>
           <xref linkend="end2xml" />
         </seg>
         <seg>convert EndNote (Refer format) to MODS</seg>
@@ -146,6 +159,13 @@
         </seg>
         <seg>convert RIS format to MODS</seg>
       </seglistitem>
+      <seglistitem>
+        <seg>
+          <xref linkend="wordbib2xml" />
+        </seg>
+        <seg>convert Word2007 bibliography XML to MODS</seg>
+      </seglistitem>
+
       </segmentedlist>
       </refsect2>
     <refsect2>
@@ -198,6 +218,28 @@
 	  byte order mark 
 	</seg>
     </seglistitem>
+      <seglistitem>
+	<seg>-x</seg>
+	<seg>--xml-entities</seg>
+	<seg>
+	  encode unicode characters as XML entities (opposite of -u)
+	</seg>
+      </seglistitem>
+      <seglistitem>
+	<seg>-nl</seg>
+	<seg>--no-latex</seg>
+	<seg>
+	  do not convert latex-style character combinations
+	</seg>
+      </seglistitem>
+      <seglistitem>
+	<seg>-s</seg>
+	<seg>--single-refperfile</seg>
+	<seg>
+	  one reference per output file
+	</seg>
+      </seglistitem>
+
     <seglistitem>
       <seg>-d</seg>
       <seg>--drop-key</seg><seg> don't put citation key in the
@@ -213,6 +255,19 @@
       </seg>
     </seglistitem>
     <seglistitem>
+      <seg>-a</seg>
+      <seg>--asis</seg><seg> with argument specifying a
+	file containing a list of names be treated verbatim
+      </seg>
+    </seglistitem>
+    <seglistitem>
+      <seg>-nt</seg>
+      <seg>--nosplit-title</seg><seg> 
+      don't split titles into TITLE/SUBTITLE pairs
+      </seg>
+    </seglistitem>
+
+    <seglistitem>
       <seg></seg><seg>--verbose</seg>
       <seg> verbose	output </seg>
     </seglistitem>
@@ -235,6 +290,15 @@
       standard input.</para>
       <programlisting>bib2xml bibtex_file.bib &gt; output_file.xml</programlisting>
     </refsect2>
+    <refsect2 id="biblatex2xml" xreflabel="biblatex2xml">
+      <title>biblatex2xml</title>
+      <para>
+      <emphasis role="bold">bib2xml</emphasis> converts a biblatex
+      (not to be confused with bibtex) formatted reference file to an
+      XML-intermediate bibliography file. 
+      .</para>
+      <programlisting>bib2xml biblatex_file.bib &gt; output_file.xml</programlisting>
+    </refsect2>
     <refsect2 id="copac2xml" xreflabel="copac2xml">
       <title>copac2xml</title>
       <para>
@@ -260,6 +324,13 @@
       <emphasis role="bold">bib2xml</emphasis>.</para>
       <programlisting>end2xml endnote_file.end &gt; output_file.xml</programlisting>
     </refsect2>
+    <refsect2 id="ebi2xml" xreflabel="ebi2xml">
+      <title>ebi2xml</title>
+      <para>
+      <emphasis role="bold">ebi2xml</emphasis> converts a EBI XML
+      file to a MODS XML-intermediate bibliography file. </para>
+    </refsect2>
+
     <refsect2 id="endx2xml" xreflabel="endx2xml">
       <title>endx2xml</title>
       <para>
@@ -292,6 +363,18 @@
       <emphasis role="bold">bib2xml</emphasis></para>
       <programlisting>ris2xml ris_file.ris &gt;	output_file.xml</programlisting>
     </refsect2>
+    <refsect2 id="wordbib2xml" xreflabel="wordbib2xml">
+      <title>wordbib2xml</title>
+      <para>
+      <emphasis role="bold">ris2xml</emphasis> converts a
+      RIS-formatted reference file to an XML-intermediate
+      bibliography file. 
+      <emphasis role="bold">ris2xml</emphasis> usage is as 
+      <emphasis role="bold">end2xml</emphasis> and 
+      <emphasis role="bold">bib2xml</emphasis></para>
+    </refsect2>
+
+
     </refsect1>
     <refsect1>
       <title>Converting from MODS</title>
@@ -364,6 +447,11 @@
 	    put one reference per file
             name by the reference number</seg>
 	</seglistitem>
+	<seglistitem>
+	  <seg>-nb</seg><seg>--no-bom</seg><seg>
+	do not write Byte Order Mark if writing UTF8</seg>
+	</seglistitem>
+
 	</segmentedlist>
       </refsect2>
     <refsect2 id="xml2bib" xreflabel="xml2bib">
--- bibutils-4.12.orig/packageconfig_start
+++ bibutils-4.12/packageconfig_start
@@ -1,10 +1,10 @@
 prefix=/usr
-exec_prefix=\${prefix}
-libdir=\${prefix}/lib
-includedir=\${prefix}/include
+exec_prefix=${prefix}
+libdir=${prefix}/lib
+includedir=${prefix}/include
 
 Name: bibutils
 Description: Converter library for various bibliography formats
 Version: VERSION
-Libs: -L\${libdir} -lbibutils
-Cflags: -I\${includedir}
+Libs: -L${libdir} -lbibutils
+Cflags: -I${includedir}
--- bibutils-4.12.orig/test/Makefile
+++ bibutils-4.12/test/Makefile
@@ -9,13 +9,13 @@ PROGS      = newstr_test entities_test u
 
 all: $(PROGS)
 
-entities_test : entities_test.o ../lib/libbibutils.a
+entities_test : entities_test.o ../lib/libbibcore.a
 	$(CC) $(LDFLAGS) $^ $(LOADLIBS) $(LDLIBS) -o $@
 
-utf8_test : utf8_test.o ../lib/libbibutils.a
+utf8_test : utf8_test.o ../lib/libbibcore.a
 	$(CC) $(LDFLAGS) $^ $(LOADLIBS) $(LDLIBS) -o $@
 
-newstr_test : newstr_test.o ../lib/libbibutils.a
+newstr_test : newstr_test.o ../lib/libbibcore.a
 	$(CC) $(LDFLAGS) $^ $(LOADLIBS) $(LDLIBS) -o $@
 
 test: entities_test newstr_test FORCE
@@ -29,4 +29,7 @@ clean:
 realclean:
 	rm -f *.o core $(PROGS)
 
+../lib/libbibcore.a:
+	make -C ../lib -f Makefile.static libbibcore.a
+
 FORCE:
--- bibutils-4.12.orig/lib/Makefile.dynamic
+++ bibutils-4.12/lib/Makefile.dynamic
@@ -27,6 +27,9 @@ BIBCORE_OBJS = $(SIMPLE_OBJS) $(NEWSTR_O
 
 BIBUTILS_OBJS = $(INPUT_OBJS) $(OUTPUT_OBJS) bibutils.o
 
+SONAME=libbibutils.so.4
+SOMINOR=12
+SOFULL=$(SONAME).$(SOMINOR)
 
 all: libbibutils.so
 
@@ -34,20 +37,21 @@ all: libbibutils.so
 	$(CC) -fPIC -c $(CFLAGS) -o $@ $<
 
 libbibutils.so: $(BIBCORE_OBJS) $(BIBUTILS_OBJS)
-	$(CC) -shared -Wl,-soname,libbibutils.so.4 -o libbibutils.so.4.12 $^
-	ln -s libbibutils.so.4.12 libbibutils.so.4
-	ln -s libbibutils.so.4.12 libbibutils.so
+	$(CC) -shared -Wl,-soname,$(SONAME) -o $(SOFULL) $^
+	ln -sf $(SOFULL) $(SONAME)
+	ln -sf $(SOFULL) libbibutils.so
 
 install:
 	echo INSTALLING LIBRARIES TO $(LIBINSTALLDIR)
-	cp libbibutils.so.4.12 $(LIBINSTALLDIR)
-	ln -sf $(LIBINSTALLDIR)/libbibutils.so.4.12 $(LIBINSTALLDIR)/libbibutils.so
-	ln -sf $(LIBINSTALLDIR)/libbibutils.so.4.12 $(LIBINSTALLDIR)/libbibutils.so.4
+	mkdir -p $(LIBINSTALLDIR)
+	cp $(SOFULL) $(LIBINSTALLDIR)
+	ln -sf $(LIBINSTALLDIR)/$(SOFULL) $(LIBINSTALLDIR)/libbibutils.so
+	ln -sf $(LIBINSTALLDIR)/$(SOFULL) $(LIBINSTALLDIR)/$(SONAME)
 
 clean: 
 	/bin/rm -f *.o core
 
 realclean:
-	/bin/rm -f *.o *.so *.so.4 *.so.4.12 core
+	/bin/rm -f *.o *.so $(SOFULL) $(SONAME) core
 
 test:
